workshops/management/commands/instructors_last_call.py

```diff
@@ -0,0 +1,32 @@
+import sys
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand, CommandError
+from workshops.models import Badge, Event, Person, Role, Tag, Task
+
+class Command(BaseCommand):
+    help = 'Create mail addresses for events about to go stale: command line is event slugs.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'event', help='Event slug',
+        )
+
+    def handle(self, *args, **options):
+
+        # Every learner who was at this event but isn't an instructor.
+        try:
+            slug = options['event']
+            event = Event.objects.get(slug=slug)
+            learner = Role.objects.get(name='learner')
+            ttt = Tag.objects.get(name='TTT')
```

Comment:
You're not using this variable, maybe you wanted to make sure that the fetched event has this tag?

Anyway, I checked the code and it's working correctly, so go ahead and merge.